app/views/assignments/show.html.erb

```diff
@@ -24,7 +24,7 @@
   <% end %>
 <% end %>
 
-<% content_for :title, "#{Assignment.model_name.human} #{@assignment.short_identifier}" %>
+<% content_for :title, "#{@assignment.short_identifier}: #{@assignment.description}" %>
```

Comment:
This is on the right track, but loses out on distinguishing the "peer review" assignments (see code you deleted that was replaced by "Assignment information". This title should be consistent with that code.